WooCommerce/Classes/ViewRelated/Products/Add Product/AddProductFromImage/AddProductFromImageViewModel.swift

```diff
@@ -42,15 +47,19 @@ final class AddProductFromImageViewModel: ObservableObject {
 
     private let addProductSource: AddProductCoordinator.Source
     private let onAddImage: (MediaPickingSource) async -> MediaPickerImage?
-    private var selectedImageSubscription: AnyCancellable?
+    private var subscriptions: Set<AnyCancellable> = []
 
     // MARK: - Scanned Texts
 
     @Published var scannedTexts: [ScannedTextViewModel] = []
+
+    @Published private var scannedTextValidation: [String: Bool] = [:]
```

Comment:
Super nit: Please consider adding a comment to explain what will be stored in this property.